apps/common/models/file-upload.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const url = require('url');
+
+const Model = require('hmpo-model');
```

Comment:
I just copied this - https://github.com/UKHomeOffice/firearms/blob/master/apps/common/models/postcodes.js#L4

Happy to change both though.